services/net/content/ContentManager.cs

```diff
@@ -446,4 +482,47 @@ private async Task<ContentModel> CopyFileWithSSHAsync(DataLocationModel dataLoca
         return result;
     }
     #endregion
+
+    #region Helper Methods
+    private static ContentTonePoolModel? GetTonePoolMapping(IEnumerable<API.Areas.Editor.Models.TonePool.TonePoolModel> tonePools, int toneValue, string? userIdentifier)
+    {
+        // use the "Default" TonePool
+        if (string.IsNullOrEmpty(userIdentifier)) {
+            // the "Default" tone should always be present
+            var tonePoolDefault = tonePools.Where(s => s.Name.Equals("Default")).First();
+            return new ContentTonePoolModel() {
+                Value = toneValue,
+                Id = tonePoolDefault.Id,
+                ContentId = 0,
+                Name = tonePoolDefault.Name,
+                OwnerId = tonePoolDefault.OwnerId
+            };
+        } else {
+            // map a specific tone pool
+            // TODO: lookup OwnerId if UserIdentifier is set
+            return null;
+        }
+    }
+
+    private static ContentTopicModel? GetTopicMapping(IEnumerable<API.Areas.Editor.Models.Topic.TopicModel> topics, string eodCategoryGroup, string eodCategory)
+    {
+        var topicModel = new ContentTopicModel() {
+            Score = 0, // TODO: until we can dig score out of TNO 1.0
+            ContentId = 0 // for new content
+        };
+        var topic = topics.Where(s => s.Name == eodCategory && s.TopicType.ToString().Equals(eodCategoryGroup,StringComparison.InvariantCultureIgnoreCase)).FirstOrDefault();
+        if (topic != null) {
+            topicModel.Id = topic.Id;
+            topicModel.Name = topic.Name;
+            topicModel.TopicType = topic.TopicType;
+        } else {
+            // KGM: will this create a new Topic, if one doesnt already exist?
```

Comment:
Yes the Content Service will need to create a topic if it doesn't exist.  However, it will have to disabled because we only want current topics enabled.